### ISSUE TITLE
Using PERSIST_PATH with empty persistence folder seems to break startup

### ISSUE THREAD

**Issue by whummer** [etspaceman/kinesis-mock#125]
Using PERSIST_PATH with empty persistence folder seems to break startup.

Example from LocalStack:
```
DATA_DIR=/tmp/datadir1 localstack start
...
[ioapp-compute-1] INFO  2021-06-21 23:02:49 k.m.KinesisMockService contextId=091765b7-d2d4-11eb-ae88-35189dcfe17a, cacheConfig={"awsAccountId":"000000000000","awsRegion":"eu-central-1","createStreamDuration":{"length":500,"unit":"MILLISECONDS"},"deleteStreamDuration":{"length":500,"unit":"MILLISECONDS"},"deregisterStreamConsumerDuration":{"length":500,"unit":"MILLISECONDS"},"initializeStreams":null,"mergeShardsDuration":{"length":500,"unit":"MILLISECONDS"},"persistConfig":{"fileName":"kinesis-data.json","interval":{"length":5,"unit":"SECONDS"},"loadIfExists":true,"path":"/tmp/ls-data1/kinesis","shouldPersist":true},"registerStreamConsumerDuration":{"length":500,"unit":"MILLISECONDS"},"shardLimit":100,"splitShardDuration":{"length":500,"unit":"MILLISECONDS"},"startStreamEncryptionDuration":{"length":500,"unit":"MILLISECONDS"},"stopStreamEncryptionDuration":{"length":500,"unit":"MILLISECONDS"},"updateShardCountDuration":{"length":500,"unit":"MILLISECONDS"}} - Logging Cache Config
os.PathError$InvalidSegment: [] is not a valid path segment. OS-Lib does not allow empty path segments If you are dealing with path-strings coming from external sources, use the Path(...) or RelPath(...) constructor calls to convert them.
at os.BasePath$.fail$1(Path.scala:120)
at os.BasePath$.checkSegment(Path.scala:137)
at os.PathChunk$StringPathChunk.<init>(Path.scala:21)
at os.PathChunk$.StringPathChunk(Path.scala:20)
at kinesis.mock.cache.PersistConfig.createPath(PersistConfig.scala:25)
at kinesis.mock.cache.PersistConfig.osPath(PersistConfig.scala:34)
at kinesis.mock.cache.Cache$.$anonfun$loadFromFile$1(Cache.scala:1245)
at ifM$extension @ org.typelevel.log4cats.slf4j.internal.Slf4jLoggerInternal$.org$typelevel$log4cats$slf4j$internal$Slf4jLoggerInternal$$contextLog(Slf4jLoggerInternal.scala:55)
at ifM$extension @ org.typelevel.log4cats.slf4j.internal.Slf4jLoggerInternal$.org$typelevel$log4cats$slf4j$internal$Slf4jLoggerInternal$$contextLog(Slf4jLoggerInternal.scala:55)
at flatMap @ kinesis.mock.KinesisMockService$.$anonfun$run$6(KinesisMockService.scala:34)
at ifM$extension @ org.typelevel.log4cats.slf4j.internal.Slf4jLoggerInternal$.org$typelevel$log4cats$slf4j$internal$Slf4jLoggerInternal$$contextLog(Slf4jLoggerInternal.scala:55)
at flatMap @ kinesis.mock.KinesisMockService$.$anonfun$run$4(KinesisMockService.scala:29)
at run @ com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
at run @ com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
at delay$extension @ pureconfig.module.catseffect.package$.loadF(package.scala:51)
at subflatMap @ pureconfig.module.catseffect.package$.loadF(package.scala:52)
at leftMap @ pureconfig.module.catseffect.package$.loadF(package.scala:53)
at rethrowT @ pureconfig.module.catseffect.package$.loadF(package.scala:54)
at flatMap @ kinesis.mock.KinesisMockService$.$anonfun$run$3(KinesisMockService.scala:27)
at flatMap @ kinesis.mock.KinesisMockService$.$anonfun$run$1(KinesisMockService.scala:26)
at run @ com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
at use @ kinesis.mock.KinesisMockService$.run(KinesisMockService.scala:24)
at run @ com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
```

@etspaceman Any ideas? Please let me know if you need any more details. Thanks!

**Comment by etspaceman** [comment]
By empty persist path, you mean `/tmp/datadir1` exists but does not contain data?

**Comment by etspaceman** [comment]
@whummer This will be resolved w/ #127.

**Comment by etspaceman** [comment]
This will be released in 0.1.2
